source/_integrations/unifiprotect.markdown

```diff
@@ -0,0 +1,86 @@
+---
+title: UniFi Protect
+description: Instructions on how to configure UniFi Protect integration by Ubiquiti.
+ha_category:
+  - Hub
+  - Camera
+ha_release: 2022.2
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_quality_scale: platinum
+ha_codeowners:
+  - '@briis'
+  - '@AngellusMortis'
+  - '@bdraco'
+ha_domain: unifiprotect
+ha_platforms:
+  - camera
+---
+
+The UniFi Protect integration, adds support for retrieving Camera feeds and Sensor data from an [UniFi Protect application](https://ui.com/camera-security) by [Ubiquiti Networks, inc.](https://www.ui.com/) that is running on an UniFi OS Console.
+
+## Prerequisites
+
+### Hardware Support
+
+This Integration supports all UniFi OS Consoles that can run UniFi Protect. Currently this includes:
+
+* UniFi Protect Network Video Recorder (**UNVR**)
+* UniFi Protect Network Video Recorder Pro (**UNVRPRO**)
+* UniFi Dream Machine Pro (**UDMP**)
+* UniFi Cloud Key Gen2 Plus (**CKGP**) firmware version v2.0.24+
+
+Ubiquiti released v2.0.24 as an official firmware release for the CloudKey+, and it is recommended that people upgrade to this UniFi OS based firmware for their CloudKey+, as this gives a much better realtime experience.
+
```

Comment:
```suggestion
```

I think this can be dropped as its been a while and they can figure this out from Ubi instead